build-tools-internal/muted-tests.yml

```diff
@@ -0,0 +1,25 @@
+tests:
```

Comment:
We have to make sure to remove these before merging... they were random for testing